web/app/services/data.service.ts

```diff
@@ -24,4 +25,10 @@ export class DataService {
     return this.http.get<ProjectResponse>(url).pipe(
         map((project) => new Project(project)));
   }
+
+  getRepos(): Observable<Repository[]> {
+    const url = `${HOSTNAME}/repos`;
```

Comment:
i like this path. I don't think it matches what we have in the controller, since it's currently `/projects/repos`. But by moving repos into it's own controller can can have `/repos` 👍